### DISCUSSION TITLE
update BC connection definition in PowerApp (canvas app)

### DISCUSSION THREAD

**Discussion by grantbrimhall** [microsoft/AL-Go#1272] [Q&A] [answered]
We have built a canvas app for a client, and need to update the connection settings to the BC environment each time we release to a higher environment.
 
I am looking at your AL-Go / Scenarios. I think it is walking me through what I need to do. Am I on the right track? Here’s what I gathered from the documentation:
 
1. SetupPowerPlatform
a. seems straightforward, no question at the moment.

2. AddPowerAppProject
a. Run the pull power platform changes action
b. Run the push power platform changes action
 
Is that it? Can all of these actions feed 100% off of al-go-settings? Or are there some things that the user must enter by hand when running the actions? (like when it asks for the environment?)
 
Client wants to do this DEV to UAT. Then again UAT to PROD. Seems like we can use the same process, just vary the answer to the environment question.
 
Thanks.

**Comment by freddydk** [comment, marked as answer]
Yes, you are on the right track.
Step 19-21 in the README here: https://github.com/microsoft/AL-Go should explain this.
